app/src/main/res/values/strings.xml

```diff
@@ -83,15 +83,15 @@
 
     <string name="off_type">Off type</string>
     <string name="annual_leave">Annual leave (14.5 days)</string>
-    <string name="leave_for_marriage">Leave for marriage (3.0 days)</string>
-    <string name="leave_for_child_marriage">Leave for child\'s marriage (1.0 days)</string>
-    <string name="funeral_leave">Funeral leave (3.0 days)</string>
-    <string name="leave_for_care_of_sick_child">Leave for care of sick child (20.0 day)</string>
-    <string name="pregnancy_examination_leave">Pregnancy examination leave (5.0 day)</string>
-    <string name="sick_leave">Sick leave (60.0 days)</string>
-    <string name="miscarriage_leave">Miscarriage leave (50.0 days)</string>
-    <string name="maternity_leave">Maternity leave (180.0 days)</string>
-    <string name="wife_labor_leave">Wife’s labor leave (14.0 days)</string>
+    <string name="leave_for_marriage">Leave for marriage (%s days)</string>
+    <string name="leave_for_child_marriage">Leave for child\'s marriage (%s days)</string>
+    <string name="funeral_leave">Funeral leave</string>
```

Comment:
WARNING: [StringFormatCount] Inconsistent number of arguments in formatting string `funeral_leave`; found both 0 and 1
 When a formatted string takes arguments, it usually needs to reference the same arguments in all translations (or all arguments if there are no translations.

There are cases where this is not the case, so this issue is a warning rather than an error by default. However, this usually happens when a language is not translated or updated correctly.